tf_agents/agents/ddpg/critic_network_test.py

```diff
@@ -1,135 +1,135 @@
-# coding=utf-8
```

Comment:
any idea why this entire file shows a diff?  its hard to see what changes you made.

Reply:
(separately; i think we should get rid of the # coding=utf-8 line)

Reply:
caused by 'autocrlf', reverted to the previous commit